### PR TITLE
add tests for SPF and DMARC

### PR DESCRIPTION
**Pull request by afeld**
Ensures that all second-level domains have valid records. Part of https://github.com/18F/tts-tech-portfolio/issues/572.

### CODE REVIEW

**Comment by afeld** [comment]
Build will fail until https://github.com/18F/dns/pull/448 is merged, as those domains don't have SPF/DMARC set up.

File: .circleci/config.yml [hunk 1]
```diff
@@ -53,6 +65,13 @@ workflows:
               only: master
           requires:
             - plan
+      - validate
+      # - validate:
+      #     filters:
+      #       branches:
+      #         only: master
+      #     requires:
+      #       - apply
```
Comment (afeld) [inline .circleci/config.yml:74]: Will change this to only run after `apply` (on `master`) so that and new domains are caught right after merge.

**Comment by amirbey** [comment]
LGTM w/ assumption https://github.com/18F/dns/pull/448 will resolve circle validation

**Comment by afeld** [comment]
[Previous build](https://app.circleci.com/pipelines/github/18F/dns/85/workflows/ce5f877c-40a6-47c1-b604-3d85f6273480/jobs/1155) was run after those old zones were deleted. Rebased to only run the validations on `master`.

**Pull request merged**